[CI] CrossClusterAsyncQueryStopIT testStopQueryLocal failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #10249 / almalinux-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/cd6yj66okzxru)
- [elasticsearch-pull-request #87730 / part-3](https://gradle-enterprise.elastic.co/s/bwhon3ygep554)
- [elasticsearch-pull-request #87581 / part-3](https://gradle-enterprise.elastic.co/s/s34geysc4m6gk)
- [elasticsearch-pull-request #87478 / part-3](https://gradle-enterprise.elastic.co/s/wuizi6p74buwm)
- [elasticsearch-pull-request #86924 / part-3](https://gradle-enterprise.elastic.co/s/yvt3wcu7ukdwi)
- [elasticsearch-pull-request #86801 / part-3](https://gradle-enterprise.elastic.co/s/knaznfts3yjyq)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.action.CrossClusterAsyncQueryStopIT.testStopQueryLocal" -Dtests.seed=17F68441DB0C201F -Dtests.locale=en-LR -Dtests.timezone=Europe/Ljubljana -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.action.CrossClusterAsyncQueryStopIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testStopQueryLocal'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: timeout waiting for DELETE response
```

**Issue Reasons:**
- [main] 6 failures in test testStopQueryLocal (0.6% fail rate in 962 executions)
- [main] 5 failures in step part-3 (1.2% fail rate in 401 executions)
- [main] 5 failures in pipeline elasticsearch-pull-request (1.3% fail rate in 399 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -411,9 +411,6 @@ tests:
 - class: org.elasticsearch.xpack.search.CrossClusterAsyncSearchIT
   method: testCancelViaExpirationOnRemoteResultsWithMinimizeRoundtrips
   issue: https://github.com/elastic/elasticsearch/issues/127302
-- class: org.elasticsearch.xpack.esql.action.CrossClusterAsyncQueryStopIT
-  method: testStopQueryLocal
-  issue: https://github.com/elastic/elasticsearch/issues/133481
 - class: org.elasticsearch.xpack.esql.inference.rerank.RerankOperatorTests
   method: testSimpleCircuitBreaking
   issue: https://github.com/elastic/elasticsearch/issues/133619

--- a/x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterAsyncQueryStopIT.java
+++ b/x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterAsyncQueryStopIT.java
@@ -8,6 +8,7 @@
 package org.elasticsearch.xpack.esql.action;
 
 import org.elasticsearch.Build;
+import org.elasticsearch.ElasticsearchTimeoutException;
 import org.elasticsearch.action.ActionFuture;
 import org.elasticsearch.client.internal.Client;
 import org.elasticsearch.core.Tuple;
@@ -16,6 +17,8 @@ import org.elasticsearch.logging.Logger;
 import org.elasticsearch.tasks.TaskInfo;
 import org.elasticsearch.transport.TransportService;
 import org.elasticsearch.xpack.core.async.AsyncStopRequest;
+import org.elasticsearch.xpack.core.async.DeleteAsyncResultRequest;
+import org.elasticsearch.xpack.core.async.TransportDeleteAsyncResultAction;
 import org.elasticsearch.xpack.esql.plugin.EsqlPlugin;
 import org.elasticsearch.xpack.esql.plugin.QueryPragmas;
 
@@ -28,7 +31,6 @@ import java.util.concurrent.atomic.AtomicInteger;
 
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
 import static org.elasticsearch.xpack.esql.action.AbstractEsqlIntegTestCase.randomIncludeCCSMetadata;
-import static org.elasticsearch.xpack.esql.action.EsqlAsyncTestUtils.deleteAsyncId;
 import static org.elasticsearch.xpack.esql.action.EsqlAsyncTestUtils.getAsyncResponse;
 import static org.elasticsearch.xpack.esql.action.EsqlAsyncTestUtils.startAsyncQuery;
 import static org.elasticsearch.xpack.esql.action.EsqlAsyncTestUtils.startAsyncQueryWithPragmas;
@@ -130,7 +132,7 @@ public class CrossClusterAsyncQueryStopIT extends AbstractCrossClusterTestCase {
         } finally {
             // Ensure proper cleanup if the test fails
             CountingPauseFieldPlugin.allowEmitting.countDown();
-            assertAcked(deleteAsyncId(client(), asyncExecutionId));
+            deleteAsyncId(client(), asyncExecutionId);
         }
     }
 
@@ -227,7 +229,7 @@ public class CrossClusterAsyncQueryStopIT extends AbstractCrossClusterTestCase {
             }
         } finally {
             SimplePauseFieldPlugin.allowEmitting.countDown();
-            assertAcked(deleteAsyncId(client, asyncExecutionId));
+            deleteAsyncId(client, asyncExecutionId);
         }
     }
 
@@ -270,7 +272,7 @@ public class CrossClusterAsyncQueryStopIT extends AbstractCrossClusterTestCase {
             }
         } finally {
             SimplePauseFieldPlugin.allowEmitting.countDown();
-            assertAcked(deleteAsyncId(client(), asyncExecutionId));
+            deleteAsyncId(client(), asyncExecutionId);
         }
     }
 
@@ -356,7 +358,16 @@ public class CrossClusterAsyncQueryStopIT extends AbstractCrossClusterTestCase {
         } finally {
             // Ensure proper cleanup if the test fails
             CountingPauseFieldPlugin.allowEmitting.countDown();
-            assertAcked(deleteAsyncId(client(), asyncExecutionId));
+            deleteAsyncId(client(), asyncExecutionId);
+        }
+    }
+
+    public void deleteAsyncId(Client client, String id) {
+        try {
+            DeleteAsyncResultRequest request = new DeleteAsyncResultRequest(id);
+            assertAcked(client.execute(TransportDeleteAsyncResultAction.TYPE, request).actionGet(30, TimeUnit.SECONDS));
+        } catch (ElasticsearchTimeoutException e) {
+            LOGGER.warn("timeout waiting for DELETE response: {}: {}", id, e);
         }
     }
 

--- a/x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterAsyncQueryStopIT.java
+++ b/x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterAsyncQueryStopIT.java
@@ -43,6 +43,11 @@ public class CrossClusterAsyncQueryStopIT extends AbstractCrossClusterTestCase {
 
     private static final Logger LOGGER = LogManager.getLogger(CrossClusterAsyncQueryStopIT.class);
 
+    @Override
+    protected boolean reuseClusters() {
+        return false;
+    }
+
     public void testStopQuery() throws Exception {
         assumeTrue("Pragma does not work in release builds", Build.current().isSnapshot());
         Map<String, Object> testClusterInfo = setupClusters(3);

